### PR TITLE
docs: add initial release process description skeleton

### PR DESCRIPTION
**Pull request by SebastianBezold**
## Description

This PR is an initial effort to formally describe our release process. 
Due to the nature of Tractus-X, the release process documentation is split in a "product-" and a "Tractus-X" release documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,86 +6,15 @@ If you are interested in the current state of planning, have a look at our [Rele
 
 ## The planning and roadmap process
 
-We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+For detailed proecess descriptions, see [the planning documentation](./docs/planning.md).
```
Comment (evegufy) [inline README.md:9]: typo in process

File: README.md [hunk 2]
```diff
@@ -6,86 +6,15 @@ If you are interested in the current state of planning, have a look at our [Rele
...
-
-Each acceptance criteria issue in GitHub contains a note with the prime contacts so that it is clear who is the assigned expert or release manager.
-
-![Happy Path](docs/static/releasemanagement-acceptance-happy-path.svg)
-
-### The other Release Path
-
-If the evidence is not sufficient so that the criterium can not be accepted in the quality gate (QG), obligations for the product team will be defined to make a reassessment.
-![The other Path](docs/static/releasemanagement-acceptance-other-path.svg)
+See [the product release documentation](./docs/product_release.md), for information about singular product releases.
```
Comment (evegufy) [inline README.md:15]: could look nicer like this:
See [product release](./docs/product_release.md), for information about singular product releases.

File: README.md [hunk 3]
```diff
@@ -6,86 +6,15 @@ If you are interested in the current state of planning, have a look at our [Rele
...
 
+See [the Tractus-X release documentation](./docs/tractus-x-release.md), for our overarching release strategy
```
Comment (evegufy) [inline README.md:17]: could look nicer like this:
See [Tractus-X release ](./docs/tractus-x-release.md), for our overarching release strategy.

File: docs/product_release.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Product Release
+
+## Artifacts
+
+- Container images
+- Helm Chart
+- GitHub release (Changelog + Sourcecode)
```
Comment (evegufy) [inline docs/product_release.md:7]: I'd put the source code in first position as it's the "main" artifact IMO

File: docs/tractus-x-release.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
```
Comment (evegufy) [inline docs/tractus-x-release.md:6]: I'd put the changelog in first position as it's the already established artifact

File: docs/product_release.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Product Release
```
Comment (evegufy) [inline docs/product_release.md:1]: Maybe it's worth mentioning that products have their release processes and to refer to the according product specific repositories for information about those.

File: docs/product_release.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+## Versioning
+
+- Semantic Versioning
+
+## Quality Assurance
+
+- Defined in TRGs
+- Responsibility of every contributor
+- (Additonally verified in quality gate process. TBD, if kept after consortia)
```
Comment (evegufy) [inline docs/product_release.md:17]: Typo in Additonally but mainly I'd not mention consortia, may just:
Currently still verified in a Quality Gate process. TBD if this process will continue.

File: docs/tractus-x-release.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+## Versioning
+
+- Semantic Versioning
+
+## Process
+
+<!-- How are products (considered to be) integrated in the Tractus-X release -->
+- Prior to release, a Tractus-X umbrella Chart release candidate branch is created
+- Producs issue PRs with their release candidate
```
Comment (evegufy) [inline docs/tractus-x-release.md:16]: This should be the future process, right? currently that's not happening and maybe that should be mentioned.

**Review by evegufy — CHANGES_REQUESTED** [review]

File: docs/planning.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+We really welcome every contribution. Every Bug report and feature proposal takes time to prepare,
+is valuable to our project, and we very much appreciate this input.
+We are giving our best to give a first feedback in one week.
+If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+
+### Issue structure
+
+Our issues do have important properties, that enable our planning process. These are:
+
+- __Labels:__ We use them to indicate the involved teams (kit or foss component). A label for each involved component is added to an issue
```
Comment (RolaH1t) [inline docs/planning.md:33]: KIT and FOSS in capitals

File: docs/planning.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+- __Issue Type:__ To separate between bugs, feature requests and release criterias, we use a custom field `Issue Type`
```
Comment (RolaH1t) [inline docs/planning.md:34]: criteria (without "s")

File: docs/planning.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+- __Milestone:__ Every Tractus-X release is represented by a `Milestone`. You can use this field to get a rough idea about the ETA
+- __Status:__ The status field is used to integrate the progress of an issue
+
+### Issue statuses
+
+The following statuses are defined:
+
+- __Inbox:__ This is the initial status of all issues. It indicates, that involved components have to be identified and additional information gathered
+- __Backlog:__ If enough information is gathered, and we agreed to work on the issue, it is set from `Inbox` to `Backlog` to indicate it is ready for timeline planning
+- __Work in Progress:__ The issue is actively been worked on.
```
Comment (RolaH1t) [inline docs/planning.md:44]: being, not been

File: docs/planning.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+
+The final status `Done` is set, as soon as all relevant implementations are done, tested and released. This has to be achieved for every change in every involved component.
+
+### Planning component changes
+
+While the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26) is used to coordinate overarching feature and bug requests,
+we encourage every component team to break these issues down to their component repositories/projects.
+When doing so, make sure you link to the overarching issue in your component issue description.
+
+## Release Management Acceptance Criteria
```
Comment (RolaH1t) [inline docs/planning.md:65]: IMHOeverything from here on is not confirmed yet

File: docs/product_release.md [hunk 13]
```diff
@@ -0,0 +1,18 @@
+# Product Release
+
+## Artifacts
+
+- Container images
+- Helm Chart
```
Comment (RolaH1t) [inline docs/product_release.md:6]: shall we call these "Product Helm Chart" => to differentiate from "Umbrella HC" in TX-Release?

File: docs/product_release.md [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+## Versioning
+
+- Semantic Versioning
+
+## Quality Assurance
+
+- Defined in TRGs
+- Responsibility of every contributor
+- (Additonally verified in quality gate process. TBD, if kept after consortia)
+
```
Comment (RolaH1t) [inline docs/product_release.md:18]: ## process abstract ##
Once planning activities for a certain iteration are completed: all contributors follow their state-of-the-art feature work on all issues in status `work in progress`. The vehicle of product helm charts shall be used to integration-test any dependencies. All valid TRGs shall be followed, as those are crosschecked by committers upon merging any pull request - leading into a flawless GitHub release per product.

File: docs/product_release.md (hunk 13 above)
Comment (RolaH1t) [reply docs/product_release.md:6]: Product Helm Chart with Dependencies

File: docs/tractus-x-release.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
+
+## Versioning
+
+- Semantic Versioning
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:10]: Calendar Versioning

File: docs/planning.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+
+The following statuses are defined:
+
+- __Inbox:__ This is the initial status of all issues. It indicates, that involved components have to be identified and additional information gathered
+- __Backlog:__ If enough information is gathered, and we agreed to work on the issue, it is set from `Inbox` to `Backlog` to indicate it is ready for timeline planning
+- __Work in Progress:__ The issue is actively been worked on.
+- __Done:__ All relevant parts have been implemented and released
+
+### Issue process
+
```
Comment (jzbmw) [inline docs/planning.md:48]: This is done by every team individually. 
Point out that this is a base, but see the product specific descriptions if they exist.

File: docs/tractus-x-release.md (hunk 8 above)
Comment (SebastianBezold) [reply docs/tractus-x-release.md:16]: Yes exactly, that's why I opened this draft PR. Jumpstarting this discussion is highly needed in my opinion.
Since this process will rely heavily on committers to support it, we need to come to a basic agreement. 
The two bullet points are just a first idea, that could also make it easy for committers to support that process, since it's a more opt-in focused one.

Right now, I don't really find the time to continue working on the draft

File: docs/planning.md (hunk 16 above)
Comment (SebastianBezold) [reply docs/planning.md:48]: Definitely good point, but the `planning.md´ file is actuall just a move of some already existing `README.md` content.
If the current wording is misleading, we should fix that in a separate PR, that can be finished sooner than the Release Process discussion.

File: docs/tractus-x-release.md [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
+## Process
+
+<!-- How are products (considered to be) integrated in the Tractus-X release -->
+- Prior to release, a Tractus-X umbrella Chart release candidate branch is created
+- Producs issue PRs with their release candidate
+
```
[the remark this answers is not in the document]
Comment (SebastianBezold) [reply docs/tractus-x-release.md:17]: Good input. I'll see if I can form that to specific sections once I finally find some time for it again. I will try to avoid some of the terms you used though. Some are not (yet) known in Eclipse Tractus-X and would need too much explanation. I.e.:
`cxOS` -> If there is nothing provided by the association that we can reference, we we would need to re-specify, which is error prone and might diverge from their definiition.
`onion rings` -> meaning? relevance? 
`carry-over versions` -> not sure what is meant with that
`boundary conditions for bundles` -> what are these conditions, who defines them, where can I read about it?

File: docs/tractus-x-release.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
```
Comment (RolaH1t) [inline docs/tractus-x-release.md:2]: suggestion: ##Intro
As mentioned in [planning.md] a Tractus-X release is scheduled quarterly. The Tractus-X project has an overarching qualification process that all official quarterly Tractus-X releases adhere to. This process applies E2E activities for testing and security (relevant release candidates in compound) that stakeholders can build on. Each component can/shall be released on demand on a higher cadence, see [product_release.md].

File: docs/planning.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
+# Release and Roadmap Process
+
+We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+
+While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
+has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
+is used to align the overarching Tractus-X releases.
+
```
Comment (HFocken) [inline docs/planning.md:8]: I suggest adding in a new paragraph:

### Release Cycle - major vs. minor release

Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release [tractus-x-release.md](url) following CalVer is released. Each Tractus-X release contains multiple products [product_release.md](url) Once per year, Tractus-X releases a major release, making the remaining three releases minor releases. A major release may contain critical breaking changes that have a major impact on data space participants,
such as changes to enablement services. A minor release contains backward compatible functionality. In addition, patch versions provide backward compatible bug and security fixes.

File: docs/tractus-x-release.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (RolaH1t) [reply docs/tractus-x-release.md:17]: cxOS may not yet be fully specified-out, but is mentioned and described by the association in the following zwo (maybe more) documents. So referencing without re-specification should be ok.
https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf)
https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_LifeCycleManagement.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_LifeCycleManagement.pdf)

[the remark this answers is not in the document]
Comment (RolaH1t) [reply docs/tractus-x-release.md:17]: the pictogram of "onion rings" shall be published near term, also via association, with regards to test management. will add the link here as soon as available. @danielmiehle any additional info?

[the remark this answers is not in the document]
Comment (RolaH1t) [reply docs/tractus-x-release.md:17]: carry-over: in every quarterly TX release, we shall expect evolutions (new versions) of some cxOS components,
but also some other components may NOT face a change/up-revision in the given iteration. Nevertheless those components are still essential parts of the bundle.
This is what I meant with "carry over versions"

File: docs/tractus-x-release.md (hunk 18 above)
Comment (SebastianBezold) [reply docs/tractus-x-release.md:2]: From my point of view, the E2E and Security part need to be clarified. 
To the best of my knowledge, the Tractus-X project does not have any running and persistent infrastructure, that is necessary to support the E2E and Security activities, that are currently mainly driven by the Catena-X consortia. 
Also, the automated e2e-testing activities are still far from making the manual steps unnecessary.

So in my opinion, we need to find some middle ground for what is actually supported by Tractus-X and what is a Process driven by other players, like the Catena-X association or others, that is maybe to a certain extend supported by Tractus-X contributors and committers

File: docs/planning.md (hunk 19 above)
Comment (SebastianBezold) [reply docs/planning.md:8]: Hi @HFocken, 
I also heard that, but actually never saw that in a tangible way. 
At least "major" and "minor" is not directly visible in CalVer numbers as it would be in SemVer. Is it always the last release of a year, that will contain breaking changes? 
How is this ensured in the planning process? Do issues from open planning also contain something that makes that transparent?

**Comment by SebastianBezold** [comment]
Hi @evegufy, @RolaH1t, @HFocken and @jzbmw 

Thanks a lot for your feedback so far. Unfortunately I do not have enough time right now to fully work on enhancing this PR.
Also, I find it difficult to encorporate some of your comments, because lines between association and Tractus-X are not clearly defined. Tractus-X is an independent project, that cannot be controlled by other entities. Some of the suggestions and also the linked PDFs do link to a different direction.

FYI @danielmiehle @giterrific @stephanbcbauer @Siegfriedk

File: docs/planning.md (hunk 9 above)
Comment (SebastianBezold) [reply docs/planning.md:33]: It is common practice, that GitHub issue label text is written in small letters. But maybe we could make them bold or italic to clarify, that actual labels are meant? Or maybe link to [available labels](https://github.com/eclipse-tractusx/sig-release/labels) directly?

File: docs/planning.md [hunk 20]
```diff
@@ -0,0 +1,81 @@
+# Release and Roadmap Process
+
+We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
```
[the remark this answers is not in the document]
Comment (SebastianBezold) [reply docs/planning.md]: Both fixed with [be873ad](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/be873ad177a7e48fb0abaf8501993e2b0ec7719d)

File: docs/planning.md (hunk 10 above)
Comment (SebastianBezold) [reply docs/planning.md]: Fixed with [be873ad](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/be873ad177a7e48fb0abaf8501993e2b0ec7719d)

File: docs/planning.md (hunk 11 above)
Comment (SebastianBezold) [reply docs/planning.md]: Fixed with [be873ad](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/be873ad177a7e48fb0abaf8501993e2b0ec7719d)

File: docs/product_release.md (hunk 13 above)
Comment (SebastianBezold) [reply docs/product_release.md:11]: Would "Helm Chart (referred to as product Chart)" be fine?. 
Both, the Tractus-X Chart, as well as single product Charts do have dependencies. 
Some do only include a Database dependency, but some also provide frontend and backend as separate Charts, that are then referenced in a product umbrella Chart

File: docs/product_release.md (hunk 14 above)
Comment (SebastianBezold) [reply docs/product_release.md:22]: Not quite sure, but AFAIK, only overarching features on the dataspace level are planned in `sig-release` and therefore are handled on a board with `work in progress` status. 
Product features, that support the dataspace feature, could (and IMO should) be planned on product repository level. So far, we do not define rules for how products need to handle issues and boards

File: docs/tractus-x-release.md (hunk 15 above)
Comment (SebastianBezold) [reply docs/tractus-x-release.md]: Fixed in [5afb19c](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/5afb19ccc4b03b3b3bfd37bd7ea5f929b3c5f273)

File: docs/planning.md (hunk 19 above)
Comment (Siegfriedk) [reply docs/planning.md:8]: Never heard of this too. Definitly something we should highlight more in our Release Notes @RolaH1t

Comment (RolaH1t) [reply docs/planning.md:8]: use text here AND on github.io page to initially introduce users to process.
add picture of phases.

File: docs/planning.md (hunk 9 above)
Comment (RolaH1t) [reply docs/planning.md:33]: fine with small letters

File: docs/planning.md [hunk 21]
```diff
@@ -0,0 +1,81 @@
...
+We are giving our best to give a first feedback in one week.
+If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+
+### Issue structure
+
+Our issues do have important properties, that enable our planning process. These are:
+
+- __Labels:__ We use them to indicate the involved teams (kit or foss component). A label for each involved component is added to an issue
+- __Issue Type:__ To separate between bugs, feature requests and release criteria, we use a custom field `Issue Type`
+- __Milestone:__ Every Tractus-X release is represented by a `Milestone`. You can use this field to get a rough idea about the ETA
```
Comment (RolaH1t) [inline docs/planning.md:35]: delete "You can use..."

File: docs/planning.md [hunk 22]
```diff
@@ -0,0 +1,81 @@
...
+
+- __Inbox:__ This is the initial status of all issues. It indicates, that involved components have to be identified and additional information gathered
+- __Backlog:__ If enough information is gathered, and we agreed to work on the issue, it is set from `Inbox` to `Backlog` to indicate it is ready for timeline planning
+- __Work in Progress:__ The issue is actively being worked on.
+- __Done:__ All relevant parts have been implemented and released
+
+### Issue process
+
+Every new feature proposal or bug report will be handled as issue in status `Inbox` initially. The alignment meetings are used to discuss the purpose and impact of the current issues.
+While in `Inbox` status, the involved components are discovered and respective `Labels` are added. If already possible, a desired `Milestone` can be set.
```
Comment (RolaH1t) [inline docs/planning.md:50]: delete "If already..."

File: docs/planning.md [hunk 23]
```diff
@@ -0,0 +1,81 @@
...
+Additionally, an `Assignee` is selected, who will coordinate efforts to solve the issue.
+
+After these details are clarified, an issue is moved to `Backlog` to open it for detailed timeline planning. In this status, discussions about a fitting `Iteration` is held.
```
Comment (RolaH1t) [inline docs/planning.md:53]: The fully refined status "backlog" is the entry point for our "open planning" ceremony.

File: docs/product_release.md (hunk 14 above)
Comment (RolaH1t) [reply docs/product_release.md:22]: update @stephanbcbauer & @RolaH1t 21-Mar:
Once planning activities for a certain iteration are completed: all contributors follow their state-of-the-art feature work on all issues in status work in progress. The vehicle of product helm charts shall be used to integration-test any dependencies. All valid TRGs must be followed, as those are crosschecked by committers upon merging any pull request - leading into a flawless GitHub release per component.

File: docs/planning.md [hunk 24]
```diff
@@ -0,0 +1,81 @@
...
+
+As soon as actual work is started in the selected iteration, the issue is set to `Work in Progress`. This is especially helpful on our project milestone views to get an overview of the release progress.
+
+The final status `Done` is set, as soon as all relevant implementations are done, tested and released. This has to be achieved for every change in every involved component.
+
```
Comment (RolaH1t) [inline docs/planning.md:58]: add new section:
###Issue hierarchy
@stephanbcbauer to formulate details => detailled features are planned and executed within the repositories of the respective component. Summary features are planned and managed on "SIG release" level...

File: docs/product_release.md (hunk 14 above)
Comment (SebastianBezold) [reply docs/product_release.md:22]: "state-of-the-art feature work" sounds a bit bloated, don't you think? What does that even mean 😁?

File: docs/tractus-x-release.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (RolaH1t) [reply docs/tractus-x-release.md:21]: text suggestion @stephanbcbauer & @RolaH1t 21-Mar:
Once feature work on product level is initiated, the quality assurance of the foreseen release package runs in parallel. The vehicle of Umbrella helm charts is used to continuously test pre-defined product bundles. Automatically and environment-less.
Reference onion rings: (add URL)

Note: In every quarterly TX release, we shall expect evolutions (new versions) of some components (see "product_release.md"). But also some other components may NOT face a change/up-revision in the given iteration. Nevertheless those components are still essential parts of the bundle.

Example for cxOS (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf (https://catena-x.net/fileadmin/_online_media_/231006_Whitepaper_Tractus-X.pdf):
A configured umbrella helm chart starts-up all relevant components. No matter if carry-over versions of a previous TX release or evolutions of the current iteration. User Journey tests (e2e) are automatically performed - customized for the respective bundle of components and within given boundary conditions.
Any findings ale looped back to the product development teams.
Product teams are obliged to fix any defects which prevent successful testing of the bundle.
A bundle is ready for Tractus-X release, once product feature work is completed for all components and e2e-testing was successful without critical findings.

Additionally, manual e2e tests may apply. Environment and execution tbd.

File: docs/product_release.md (hunk 14 above)
Comment (SebastianBezold) [reply docs/product_release.md]: @RolaH1t and @stephanbcbauer,
I took your input and did elaborate an change wording. What do you think of the description in this commit
[c33fcc8](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/c33fcc82d867d7bbd776cd09e20130344b0129da)?

File: docs/tractus-x-release.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (SebastianBezold) [reply docs/tractus-x-release.md:34]: I get the idea of this description, but to me it almost seems like this is talking a different language, than I would expect from this process definition. 
In commit [c33fcc8](https://github.com/eclipse-tractusx/sig-release/pull/519/commits/c33fcc82d867d7bbd776cd09e20130344b0129da) I started a process definition, that I think could make sense. It is not finished though and is also just a proposal, that we must discuss in the comitter round.

As general suggestion for wording, I would avoid terms, that are unknown, undefined or not commonly used.
I.e. carry-over, evolution, up-revision.
I would also the "what is NOT happening" statements. Instead of "maybe there are no changes, but we still need it", I would just state, that we are testing all components in the version, that is planned to be included in the release

File: docs/product_release.md [hunk 25]
```diff
@@ -0,0 +1,44 @@
+# Product Release
+
+Tractus-X products do follow their own release process. Please check the leading product repositories for details.
+Independent of the technical details of product release processes, the released artifacts and minimum level of quality ensurance
```
Comment (RolaH1t) [inline docs/product_release.md:4]: assurance

File: docs/product_release.md [hunk 26]
```diff
@@ -0,0 +1,44 @@
...
+is the same across all of them.
+
+The release contents for individual products are are planned in the [open refinement and planning](./open-refinement-and-planning.md) sessions.
```
Comment (RolaH1t) [inline docs/product_release.md:7]: are are => are

File: docs/product_release.md [hunk 27]
```diff
@@ -0,0 +1,44 @@
...
+These sessions define the scope for all dataspace components, but therefore paint a clear picture of enhancements necessary for every single product.
```
Comment (RolaH1t) [inline docs/product_release.md:8]: "and" instead of "but"

File: docs/product_release.md [hunk 28]
```diff
@@ -0,0 +1,44 @@
...
+The product teams themselfes are then responsible to define fine-grained task breakdowns as suiting their own needs.
```
Comment (RolaH1t) [inline docs/product_release.md:9]: themselves

File: docs/product_release.md [hunk 29]
```diff
@@ -0,0 +1,44 @@
...
+
+> __HINT:__
+> If product enhancements contribute to an overarching dataspace feature, please link the dataspace feature defined in `sig-release` in your product issue.
+> These links can be done in either the description or any comment on your issue. Use `eclipse-tractusx/sig-release#issue-number`
+
+Once planning activities for a certain iteration are completed, the implementation phase is starting and product teams can coordinate, where necessary
+through our [communication channels](https://eclipse-tractusx.github.io/docs/oss/getting-started#communication-channels).
+
+Integration tests are performed by maintaining an umbrella Chart, that deploys all relevant dataspace components to spin up an artificial dataspace based on Tractus-X components.
```
Comment (RolaH1t) [inline docs/product_release.md:18]: is "Product Umbrella Chart" now the best term based on conversation below?
it would be separated from " Tractus-X Umbrella Chart", right?

File: docs/product_release.md [hunk 30]
```diff
@@ -0,0 +1,44 @@
...
+There is a dedicated repository [eclipse-tractusx/tractus-x-umbrella](https://github.com/eclipse-tractusx/tractus-x-umbrella), that is maintained collectively to reflect the latest product release versions.
+
+Additional to the integration- and unit-test suites the teams maintain, every product also needs to adhere to the [Tractus-X release guidelines (TRGs)](https://eclipse-tractusx.github.io/docs/release).
```
Comment (RolaH1t) [inline docs/product_release.md:21]: Proposal for line 21: Every product needs to adhere...
Proposal to insert between 24 & 25: this is in addition to...

File: docs/product_release.md [hunk 31]
```diff
@@ -0,0 +1,44 @@
...
+Compliance to these guidelines __should__ be verified on every PR, but __must__ be complied with, as soon as a new product version is planned to be included in the Tractus-X release.
+
+> __HINT:__
+> Take special care about legal requirements defined in TRG 7. They __must__ be adhered to on every PR.
+
+Product releases are done by creating a GitHub release. Every release is following the [semantic versioning](https://semver.org/) pattern and marked in the git history via tag.
```
Comment (RolaH1t) [inline docs/product_release.md:27]: and _must_ be marked...

**Review by ghost — CHANGES_REQUESTED** [review]
I like and support the overall text proposal

**Comment by Siegfriedk** [comment]
Please be aware that we created a new PR https://github.com/eclipse-tractusx/sig-release/pull/662
